explore/src/main/scala/explore/Routing.scala

```diff
@@ -102,20 +116,25 @@ object Routing:
 
   private def proposalTab(page: Page, model: View[RootModel]): VdomElement =
     val routingInfo = RoutingInfo.from(page)
-    ProposalTabContents(
-      routingInfo.programId,
-      model.zoom(RootModel.user).get,
-      model.zoom(RootModel.undoStacks).zoom(ModelUndoStacks.forProposal)
+    // we don't need the summaries, but we still want to validate the progam id
+    withProgramSummaries(routingInfo.programId.some, model)(_ =>
+      ProposalTabContents(
+        routingInfo.programId,
+        model.zoom(RootModel.user).get,
+        model.zoom(RootModel.undoStacks).zoom(ModelUndoStacks.forProposal)
+      )
     )
 
-  private def configurationsTab(page: Page): VdomElement =
-    SequenceEditor(RoutingInfo.from(page).programId)
+  private def configurationsTab(page: Page, model: View[RootModel]): VdomElement =
+    val routingInfo = RoutingInfo.from(page)
+    // we don't need the summaries, but we still want to validate the progam id
+    withProgramSummaries(routingInfo.programId.some, model)(_ =>
+      SequenceEditor(routingInfo.programId)
+    )
 
   private def showProgramSelectionPopup(model: View[RootModel]): VdomElement =
-    ProgramsPopup(
-      currentProgramId = none,
-      undoStacks = model.zoom(RootModel.undoStacks)
-    )
+    // Because we are not supplying a program id, the ProgramsPopup will be displayed
+    withProgramSummaries(none, model)(_ => <.div("Programmer error!"))
```

Comment:
Cool, I'm looking forward to the first faul report with "Programmer error"

Reply:
Maybe I should add my name? 😆